docs/shrc_diag_util.md

```diff
@@ -262,106 +282,170 @@ Line number information for classes in the cache is then shown with one of the f
 
 The following summary data is displayed:
 
-#### `baseAddress` and `endAddress`
- : The boundary addresses of the shared memory area that contains the classes.
-
-#### `allocation pointer`
- : The address where `ROMClass` data is currently being allocated in the cache.
-
-#### `cache layer`
-: The layer number that the cache stats relate to.
-
-#### `cache size` and `free bytes`
- : `cache size` shows the total size of the shared memory area in bytes, and `free bytes` shows the free bytes that remain.
+- `baseAddress` and `endAddress`
+    - The boundary addresses of the shared memory area that contains the classes.
```

Comment:
These addresses vary from run to run, depending where the Operation System allocates the memory.